### DISCUSSION TITLE
Duplicate queries with Form Builder repeater input

### DISCUSSION THREAD

**Discussion by DanjBethel** [filamentphp/filament#13007] [Help]
### Package

Form builder

### Package Version

3.2

### How can we help you?

Not sure if I'm doing something wrong but I'm noticing a lot of duplicate queries when using the repeater. Is this just how it works? Or is there a better way to optimize this?

![Screenshot 2024-05-30 at 3 45 14 AM](https://github.com/filamentphp/filament/assets/4720302/1cc71ee6-0622-4220-b612-d53c7ebd11b8)

**Comment by DanjBethel** [comment]
Hopefully I can get a response here, still having same issue

**Reply by krekas** [reply]
try to modify query to eager load your relations
